Sources/TuistGenerator/Utils/SwiftPackageManagerInteractor.swift

```diff
@@ -47,8 +47,8 @@ public class SwiftPackageManagerInteractor: SwiftPackageManagerInteracting {
         config: Config,
         graphTraverser: GraphTraversing
     ) async throws {
-        guard !config.generationOptions.disablePackageVersionLocking,
-              graphTraverser.hasRemotePackages
```

Comment:
I could not see how the side effects by removing this check

Reply:
won't this try to install packages even if there are none defined? I think this check was here for a reason but we can run install if there is a remote _or_ a local package

Reply:
I just added check if there is package deps.
seems it's fine
https://github.com/tuist/tuist/pull/5614/commits/07237f47f3a760983f44a8aa3b4eb82c194888ac#diff-7e65e9edc17f330d07a575ad33674f3b60d1b8d190a84aec9f8d616ebfe01d86R52

Reply:
OK, I'll try to add that.